.github/workflows/mac.xml

```diff
@@ -0,0 +1,64 @@
+name: Angle3
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: ${{ matrix.target }}
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: ["mac"]
+    env:
+      # "If you are a non-googler you need to set DEPOT_TOOLS_WIN_TOOLCHAIN=0"
+      DEPOT_TOOLS_WIN_TOOLCHAIN: 0
+      # Bump this to fetch a newer version
+      ANGLE_VERSION: 4134
+    steps:
+    - uses: actions/checkout@v1
+    - name: Clone
+      run: |
+        git clone --depth=1 --branch chromium/${{ env.ANGLE_VERSION }} https://chromium.googlesource.com/angle/angle
+    - name: Set up Python 2.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 2.7
+    - name: Dependencies
+      run: |
+        Invoke-WebRequest https://storage.googleapis.com/chrome-infra/depot_tools.zip -O depot_tools.zip
+        7z x depot_tools.zip -o${{ github.workspace }}/depot_tools
+        echo "::add-path::${{ github.workspace }}\depot_tools"
+    - name: Configure
+      run: |
+        mkdir angle\out
+        mkdir angle\out\Release
+        copy args-${{ matrix.target }}.gn angle\out\Release\args.gn
+    - name: Generate
+      run: |
+        cd angle
+        python scripts/bootstrap.py
+        gclient.bat sync
+        git checkout master
+        gn gen out/Release
+    - name: Build
+      run: |
+        cd angle
+        autoninja -C out/Release libEGL libGLESv2
+    - name: Install
+      run: |
+        mkdir install
+        mkdir install\lib
+        mkdir install\bin
+        mkdir install\include
+
+        copy angle\out\Release\d3dcompiler_47.dll install\bin\
+        copy angle\out\Release\libGLESv2.dll.lib install\lib\libGLESv2.lib
+        copy angle\out\Release\libGLESv2.dll install\bin\
+        copy angle\out\Release\libEGL.dll.lib install\lib\libEGL.lib
+        copy angle\out\Release\libEGL.dll install\bin\
+        xcopy /e angle\include\* install\include\
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: angle-${{ env.ANGLE_VERSION }}-${{ matrix.target }}
+        path: install
```

Comment:
Seems that the whole workflow gets ignored if there's some error. I'd remove all this and add just

```yml
    steps:
    - name: hello
      run: |
        echo hello
```

at first, then expanding it further

Reply:
Hmm... I've tried updating the file to a more minimal version and I still can't seem to see  it reflected in this PR...

Reply:
Not here (I suppose because it runs only the workflows that are already on upstream), but I see it here: https://github.com/hsdk123/magnum-ci/actions/runs/156944912

Reply:
Change the name back to Angle so it's together with the windows jobs?

Reply:
This might sound a bit weird, but it looks as if you need to
```
a. go into actions
b. create a new workflow file through there
```
to be able to start getting that workflow file to trigger.
If you just upload a new .yml file to the repo, it seems it doesn't get registered.

Would you happen to be able to create a `mac.yml` file in the repo?
I'll try just editing that file and see what happens.

Reply:
Heh, this is absolutely weird. I refuse to accept this is how I'm supposed to things here :D

@Squareys do you remember how did you manage to create / register the Windows workflow in #1? Did you need to go through some weird steps?

Reply:
The workflow run says "This commit does not belong to any branch on this repository."
I don't know, your comment:

> Ok so maybe you need to enable https://github.com/hsdk123/magnum-ci/actions on your fork and it'll appear there?

... sounded pretty sane to me. I think initially you need to run this through your own pipeline until it's merged. Would make sense, otherwise you could go around and steal people's CI minutes :D

Reply:
> Would make sense, otherwise you could go around and steal people's CI minutes :D

Makes sense, although in that case, I'm not sure what I should be doing now. It seems the only way I can get a workflow to trigger on even my fork is to add the yml through the action tab, and if I do so, it only allows me to add the file in either the master branch, or a completely new branch, thus not the angle branch which we're working on. 

This all seems terribly roundabout to me.